CMakeLists.txt

```diff
@@ -123,9 +123,9 @@ endif()
 # FIXME: the version number should be automatically integrated
 set(CURRENT_VERSION "Uncrustify-0.67_f")
 
+find_package(PythonInterp)
```

Comment:
I think we can set that to `REQUIRED` now and omit all the found checks.

Reply:
Ack, considering it's already needed to generate the punctuator table, that's a good point. (Which is partly my fault → #1777 :flushed:.)